collab-service/src/redis_adapter/redis_question_adapter.ts

```diff
@@ -28,11 +30,29 @@ async function setQuestionRedis(
 async function getQuestionRedis(
   key: string,
   publisher: RedisClientType,
-): Promise<string | null> {
-  return publisher.get(`${redisPrefix}-${key}`);
+): Promise<string> {
+  const qns = await publisher.get(`${REDIS_PREFIX}-${key}`);
+  if (qns) {
+    return qns;
+  }
+  Logger.error(`No question of room ${key} found`);
+  return '';
+}
+
+/**
+ * Refreshes expiration of Redis question of given key
+ * @param key
+ * @param publisher
+ */
+async function heartbeatQuestionRedis(
```

Comment:
I suggest a better name.
```suggestion
async function refreshRedisQuestionExpiry(
```